deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -218,4 +337,4 @@ spec:
   minKubeVersion: 1.19.0
   provider:
     name: Red Hat
-  version: 0.0.0
+  version: 1.0.0-rc1
```

Comment:
version should be 0.0.0 here, this will be later replaced through render-templates in WMCO bundle dist-git.